src/Microsoft.FeatureManagement/FeatureManagementBuilderExtensions.cs

```diff
@@ -8,6 +8,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection;
```

Comment:
I don't think this is used.

Reply:
Oh, my bad. I forget to remove it. Good catch.